### ISSUE TITLE
Google search not using title, description tags

### ISSUE THREAD

**Issue by mpli** [vinaygopinath/ngMeta#35]
I am using Angular JS 1.58 and I am trying to change Title, Description, Keywords, dynamically using ngMeta. I successfully implemented it, no errors, and I verified that title and meta description and keywords are changing correctly, using the Chrome inspector elements.

From experimenting last several weeks, I found the following:

When I use the ngMeta to change the title and description, google search results do not pick them up and uses the page content to display a title and description. So, the content is rendered correctly, not the tags.

When I use the ngMeta to change the title only, but hard code the meta description and keywords, Google search displays the correct title and description. I verified this behavior several times, going back and forth and using the google webmaster fetch and request to index feature.

Anyone can help? Would rewriting my web site with Angular 2 resolve this issue, or since ngMeta still would need to be used, it would have the same issue?

**Comment by nicolemamay** [comment]
I am having this same issue. Have you found a fix?

**Comment by mpli** [comment]
No, there was no way doing it under Angular 1.x. I guess, it would be possible with some pre-rendering extension.
I ended up having someone rewriting my web site using Angular 2.0 Universal. It has a built in pre-rendering. Now it works great and my google search position improved greatly.
